relayer/src/services/relayer/optimism.ts

```diff
@@ -88,6 +88,7 @@ export class OptimismRelayerService extends BaseRelayerService {
   }
 
   private async safeRelayMessage(message: CrossChainMessage): Promise<boolean> {
+    const ESTIMATION_CONTRACT_ADDRESS = "0x0000000000000000000000000000000000000001";
```

Comment:
Maybe add a doc comment to the optimism contract where they expect this address for estimating gas.

Reply:
👍